lib/Importer/TFLiteModelLoader.cpp

```diff
@@ -257,7 +266,17 @@ TFLiteModelLoader::getTensorByIndex(size_t index) {
 }
 
 std::string TFLiteModelLoader::getTensorName(const tflite::Tensor *tensor) {
-  return tensor->name()->str();
+  if (tfliteRenameTensorsOpt) {
+    auto *tensors = graph_->tensors();
+    auto tensorIt = std::find(tensors->begin(), tensors->end(), tensor);
+    assert(tensorIt != tensors->end() && "Tensor not found!");
```

Comment:
Not sure if this is important to you, but `assert` will be removed if you build in optimized mode.

Reply:
Yeah I know but that is ok because normally those should never happen.